core/zms/src/main/rdl/Domain.rdli

```diff
@@ -319,4 +319,20 @@ resource AuthHistoryDependencies GET "/domain/{domainName}/history/auth" {
         ResourceError UNAUTHORIZED;
         ResourceError TOO_MANY_REQUESTS;
     }
+}
+
+type ExpiredMembers Struct {
+    Array<MemberName> expiredRoleMembers; //list of deleted expired role members names
+    Array<GroupMemberName> expiredGroupMembers; //list of deleted expired groups members names
+}
+
+// Delete expired principals
+resource ExpiredMembers DELETE "/expired-members?purgeResources={purgeResources}" {
+    Int32 purgeResources (optional); //defining which resources will be purged. by default all resources will be purged
```

Comment:
0 - none of them will be purged
1 - only roles will be purged
2 - only groups will be purged
3/null - both of them will be purged